src/test/java/org/broadinstitute/consent/http/service/MatchServiceTest.java

```diff
@@ -300,6 +301,15 @@ public void testRemoveMatchesForConsent() throws Exception {
         service.removeMatchesForConsent(sampleConsent1.getConsentId());
     }
 
+    @Test
+    public void testFindMatchesForPurposeIds() {
+        Match mockMatch = new Match();
+        when(matchDAO.findMatchesForPurposeIds(anyList()))
+            .thenReturn(List.of(mockMatch));
+        initService();
+        service.findMatchesForPurposeIds(List.of("test"));
```

Comment:
Are we actually testing anything here? Maybe wrap the service call in a try-catch and fail in the catch block?